core-common/src/main/java/org/glassfish/jersey/logging/LoggingInterceptor.java

```diff
@@ -285,6 +285,24 @@ public void write(final int i) throws IOException {
             }
             out.write(i);
         }
+
+        @Override
+        public void write(byte[] ba, int off, int len) throws IOException {
+            if ((off | len | ba.length - (len + off) | off + len) < 0) {
+                throw new IndexOutOfBoundsException();
+            } else {
+                if ((baos.size() + len) <= maxEntitySize) {
+                    for (int i = 0; i < len; ++i) {
+                        baos.write(ba[off + i]);
+                        out.write(ba[off + i]);
+                    }
+                } else {
+                    for (int i = 0; i < len; ++i) {
+                        out.write(ba[off + i]);
+                    }
+                }
+            }
+        }
```

Comment:
is there any objection to rewrite it like this:
```
        @Override
        public void write(byte[] ba, int off, int len) throws IOException {
            if ((off | len | ba.length - (len + off) | off + len) < 0) {
                throw new IndexOutOfBoundsException();
            }
            if ((baos.size() + len) <= maxEntitySize) {
                baos.write(ba, off, len);
            } 
            out.write(ba, off, len);
        }
```
?

Reply:
No objection, thanks for the tips !